docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,261 @@
-# Getting started
+# Getting started with Pebble
 
-TODO
+In this tutorial, we will download and install Pebble, configure layers, run the Pebble daemon, and work with layers and services. This tutorial takes about 15 minutes to complete.
+
+After this tutorial, you will have a basic understanding of what Pebble is and how to use it to orchestrate services, and you can continue exploring more advanced features and use cases (see {ref}`next_steps`).
+
+## Prerequisites
+
+- A Linux machine.
+
+## Download and install Pebble
+
+Get the latest tag on the [latest release page](https://github.com/canonical/pebble/releases/latest), then run the following commands to download, extract, and install the latest release (replace `v1.12.0` with the latest tag and `amd64` with your architecture):
+
+```bash
+wget https://github.com/canonical/pebble/releases/download/v1.12.0/pebble_v1.12.0_linux_amd64.tar.gz
+tar zxvf pebble_v1.12.0_linux_amd64.tar.gz
+sudo mv pebble /usr/local/bin/
+```
+
+Also, make sure `/usr/local/bin/` is in your `$PATH`.
```

Comment:
Does this need to be said? Wouldn't `/usr/local/bin` always be on `PATH` on Linux?

If we do keep it, let's drop the trailing slash after `bin/`, as that's not typical for `PATH`.

Reply:
Since the default PATH can be modified by users, I'll keep this (suggested by Anne) and remove the trailing slash.